netdisco/discoverables/logitech_mediaserver.py

```diff
@@ -11,4 +11,5 @@ def __init__(self, netdis):
 
     def get_entries(self):
         """Get all the Logitech Media Server details."""
-        return [entry['from'] for entry in self.netdis.lms.entries]
+        return [(entry['from'], entry['port'])
```

Comment:
That dict contains "data", the value of which is unclear to me. The address of the server also has the key 'from' rather than e.g. 'server', which does not seem ideal for a wider exposure. Ideally, this name should be standardized across different discovered types.

I recommend renaming "from" to "server" and check if the "data" field is needed. If not, it should be removed. Otherwise, maybe it should be filtered out from the returned dict?